db/migrate/20250109205930_set_statement_default_reconcile_amount.rb

```diff
@@ -0,0 +1,5 @@
+class SetStatementDefaultReconcileAmount < ActiveRecord::Migration[7.1]
+  def change
+    change_column_default :statements, :reconcile_amount, from: nil, to: 0
```

Comment:
Pretty against this one - we shouldn't have a 'default amount' for a number we should be calculating - if its 0 then some code should have calculated its zero and set it?

`nil` shows nothing has calculated the amount